apps/server/src/modules/sharing/uc/share-token.uc.ts

```diff
@@ -163,6 +171,21 @@ export class ShareTokenUC {
 		});
 	}
 
+	private copyColumnBoard(
+		userId: string,
+		originalColumnBoardId: string,
+		courseId: string
```

Comment:
I would recommend to rename the function parameters for all methods in this case like:
```typescript
private copyColumnBoard(
		userId: string,
		sourceColumnBoardId: string,
		targetCourseId: string
	)...
```